src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,10 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo comment={comment} />
```

Comment:
I'm not sure, but i think that's important to use 'key' on this element also, same as we did on PostInfo)